src/pages/challenges.tsx

```diff
@@ -1,17 +1,52 @@
-import { FC } from 'react';
+import { FC, useState, useEffect } from 'react';
 import Head from '@/components/head';
 import Layout from '@/components/Layout';
 import Section from '@/components/section';
 import { CHALLENGES_SCREEN } from '@/components/constants/display-sections.js';
-import ChallengesJson from '@/mocks/challenges.json';
+import useFetch from '@/hooks/useFetch';
+import classNames from '@/styles/tasks.module.scss';
 
-const Challenges: FC = () => (
-  <Layout>
-    <Head title="Challenges" />
-    <div>
-      <Section content={ChallengesJson} screen={CHALLENGES_SCREEN} />
-    </div>
-  </Layout>
-);
+const CHALLENGES_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/challenges`;
+
+const Challenges: FC = () => {
+  let challenges:any[] = [];
```

Comment:
why are we giving this `any`? don't we know yet what type of data will be stored in the array?